README.md

```diff
@@ -29,7 +29,7 @@ The rules for merging depend on the type of change in question and its scope of
 
 ## Formatting Guidelines
 
-Design documents should be placed in `docs/`.
+Design documents should be placed in [docs/](./docs). API Specifications should be placed in [api-definitions/](./api-definitions).
```

Comment:
Does `api-specifications` as directory name read better than `api-definitions`?

Reply:
I think either word works here, so no strong opinion at all

Reply:
Given my lack of opinion on this, I thought I would look to see if the internet had a strong opinion, and it turns out that `OpenAPI` itself does ([here](https://swagger.io/resources/articles/difference-between-api-documentation-specification/))!

**TLDR;** What we're discussing here is an `API Definition` (aimed at being consumed by machines) not an `API Specification` (aimed at being consumed by humans).

Reply:
>An API definition is similar to an API specification in that it provides an understanding of how an API is organized and how the API functions. But the API definition is aimed at machine consumption instead of human consumption of APIs. An API definition provides information about how the API functions, how it links with other APIs, and the expected results in a machine-readable format. It focuses on defining the API and outlining the structure of the API.

Reply:
@leseb probably does. This doc is a definition and the repo is a specification?! :)